dev/com.ibm.ws.transport.http/src/io/openliberty/http/netty/stream/WsByteBufferChunkedInput.java

```diff
@@ -0,0 +1,116 @@
+/*******************************************************************************
```

Comment:
Is this just a copy of WsByteBufferChunkedInput as a starting point hence the copyright statement? Then you've made some changes?

Reply:
Talked with Manuel and that is the case.